### PR TITLE
Pull images before updating Docker Swarm stack

### PR DESCRIPTION
**Pull request by rikukissa**
As part of deploying our environment, we noticed that even though docker stack update had gone through ok, some containers were still not running but instead were shown in `0/1` state. 
image

We realised this was to do with docker swarm still downloading the images, but even after waiting until the next morning, some of the services were not running. This PR now makes it so that images are first downloaded before the stack starts. This gives the person deploying much clearer feedback of what's happening, and prevents swarm going into this odd state.

### CODE REVIEW

File: infrastructure/server-setup/playbook-1.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@
 
     - name: 'Add docker repository'
       apt_repository:
-        repo: 'deb [arch=amd64] https://download.docker.com/linux/ubuntu {{ ansible_lsb.codename }} edge'
```
Comment (rikukissa) [inline infrastructure/server-setup/playbook-1.yml:32]: We had to change this so the docker image was found

**Review by euanmillar — APPROVED** [review]

**Comment by rikukissa** [comment]
I'm blocking this because I still need to amend this..

File: packages/config/src/config/constants.ts [hunk 2]
```diff
@@ -11,7 +11,7 @@
  */
 export const OPENCRVS_SPECIFICATION_URL = 'http://opencrvs.org/specs/'
 export const HOST = process.env.HOST || 'localhost'
-export const HOSTNAME = process.env.HOSTNAME || '*'
+export const HOSTNAME = process.env.DOMAIN || '*'
```
Comment (euanmillar) [inline packages/config/src/config/constants.ts:14]: Why do we need to change this?

**Comment by euanmillar** [comment]
Merged into https://github.com/opencrvs/opencrvs-core/pull/3921

**Pull request closed without merging**